boards/arm/usb_kw24d512/doc/usb_kw24d512.rst

```diff
@@ -0,0 +1,127 @@
++-----------+------------+-------------------------------------+
+| SPI       | on-chip    | spi                                 |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+| FLASH     | on-chip    | soc flash                           |
++-----------+------------+-------------------------------------+
+
+The default configuration can be found in the defconfig file:
+
+	``boards/arm/usb_kw24d512/usb_kw24d512_defconfig``
+
+Other hardware features are not currently supported by the port.
+
+Connections and IOs
+===================
+
+The KW2xD SoC has five pairs of pinmux/gpio controllers.
+
++-------+-----------------+--------------------------------------+
+| Name  | Function        | Usage                                |
++=======+=================+======================================+
+| PTA1  | UART0_RX        | UART Console                         |
++-------+-----------------+--------------------------------------+
+| PTA2  | UART0_TX        | UART Console                         |
++-------+-----------------+--------------------------------------+
+| PTC4  | GPIO            | SW1                                  |
++-------+-----------------+--------------------------------------+
+| PTD4  | GPIO            | Blue LED (D2)                        |
++-------+-----------------+--------------------------------------+
+| PTD5  | GPIO            | Blue LED (D3)                        |
++-------+-----------------+--------------------------------------+
+| PTB10 | SPI1_PCS0       | internal connected to MCR20A         |
++-------+-----------------+--------------------------------------+
+| PTB11 | SPI1_SCK        | internal connected to MCR20A         |
++-------+-----------------+--------------------------------------+
+| PTB16 | SPI1_SOUT       | internal connected to MCR20A         |
++-------+-----------------+--------------------------------------+
+| PTB17 | SPI1_SIN        | internal connected to MCR20A         |
++-------+-----------------+--------------------------------------+
+| PTB19 | GPIO            | internal connected to MCR20A (Reset) |
++-------+-----------------+--------------------------------------+
+| PTB3  | GPIO            | internal connected to MCR20A (IRQ_B) |
++-------+-----------------+--------------------------------------+
+| PTC0  | GPIO            | internal connected to MCR20A (GPIO5) |
++-------+-----------------+--------------------------------------+
+
+System Clock
+============
+
+The KW2xD SoC is configured to use the 4 MHz external clock from the modem
+with the on-chip PLL to generate a 48 MHz system clock.
+
+Serial Port
+===========
+
+The KW2xD SoC has six UARTs. One is configured and can be used for the console,
+the remaining are not used.
+
```

Comment:
I have left it consciously :smile:. KW24D512 is functional equal to the KW22D512, KW22D512 is supported by OpenOCD and for the KW24D512 I have a 3 lines patch. Phytec has the similar hardware, but with the KW22D512. And it is possible to flash it with jlink, so it would be a little longer chapter. Currently I do not know how I describe it best, so I would like to do it later.